meson_options.txt

```
@@ -43,8 +43,8 @@ option('enable-fp16', type: 'boolean', value: false)
 option('enable-cublas', type: 'boolean', value: false)
 option('enable-openmp', type: 'boolean', value: true)
 option('enable-opencl', type: 'boolean', value: false)
-option('enable-biqgemm', type: 'boolean', value: false)
-option('biqgemm-path', type: 'string', value: '../BiQGEMM')
+option('enable-biqgemm', type: 'boolean', value: true)
+option('biqgemm-path', type: 'string', value: '/home/donghak/workspace/AIP_nntrainer/BiQGEMM')
```

Your code has hard coded path.

BiQGEMM exists at internal repository.
So we must add conditional check for it is internal or not.

```suggestion
option('biqgemm-path', type: 'string', value: '../BiQGEMM')
```

---

That's correct. Since this is a draft, the pull request is not yet ready for review, and there are many things to organize, such as logs. Once it is ready for review, I will remove the draft status. Until then, please hold off on reviewing it!

anyway Thank you for Review 👍